components/User/ProfileModel.tsx

```diff
@@ -123,14 +148,24 @@ export const ProfileModel: FC<Props> = ({ onClose }) => {
                           ))}
                         </div>
                         {user?.plan === 'free' && (
-                          <a
-                            href={upgradeLink()}
-                            target="_blank"
-                            rel="noreferrer"
-                            className="px-4 py-2 border rounded-lg bg-neutral-100 shadow border-neutral-500 text-neutral-700 hover:bg-white focus:outline-none mt-4 text-center text-sm"
-                          >
-                            {t('Upgrade')}
-                          </a>
```

Comment:
Seems like this is why we see 3 events triggered on GA. Because this function is run every time this component is being rendered @patel-rushi